lib/ibm_watson/speech_to_text_v1.rb

```diff
@@ -530,10 +562,13 @@ def recognize_using_websocket(
         "smart_formatting" => smart_formatting,
         "speaker_labels" => speaker_labels,
         "grammar_name" => grammar_name,
-        "redaction" => redaction
+        "redaction" => redaction,
+        "processing_metrics" => processing_metrics,
+        "processing_metrics_interval" => processing_metrics_interval,
+        "audio_metrics" => audio_metrics
       }
       options.delete_if { |_, v| v.nil? }
-      WebSocketClient.new(audio: audio, chunk_data: chunk_data, options: options, recognize_callback: recognize_callback, url: url, headers: headers, disable_ssl_verification: @disable_ssl_verification)
+      WebSocketClient.new(audio: audio, chunk_data: chunk_data, options: options, recognize_callback: recognize_callback, url: url, headers: headers)
```

Comment:
Do we need to have `disableSslVerification` here?

Reply:
yeah